### ISSUE TITLE
Application file preview

### ISSUE THREAD

**Issue by ronaldoreisdesignn** [files-community/Files#6792]
### What's the Problem?

File preview is not always displayed

![Captura de tela 2021-11-07 210209](https://user-images.githubusercontent.com/93889137/140667090-7b3bf000-3153-4b0f-a805-d38cdc454c39.png)

### Solution/Idea

It's not really an idea, it's something that MacOS users already enjoy. Come on, my name is Ronaldo, I'm a graphic designer and something that I find very productive in my work is being able to have a preview of the photoshop and illustrator file, as well as in MacOS, something that Windows users don't have this experience, requiring installation of third-party programs to acquire this functionality.

I even used these tools, which is SageThumbs, but with the files I believe it is suffering some incompatibility, some files are not being done in pre-view.

In conclusion, if you've thought of something like this or if you find my feedback interesting, I and other designers will be super happy. :)

### Alternatives

Alternativa que uso é o SageThumbs

### Priorities

| Capability | Priority |
| :---------- | :------- |
| This proposal will allow developers to accomplish W | Must |
| This proposal will allow end users to accomplish X | Should |
| This proposal will allow developers to accomplish Y | Could |
| This proposal will allow end users to accomplish Z | Won't |

### Files Version

Versão: 2.0.34.0 OS Version: 10.0.22000.282

### Windows Version

 Version 21H2 Installed on ‎25/‎07/‎2021 OS Build 22000.282 Experience Windows Feature Experience Pack 1000.22000.318.0

**Comment by gave92** [comment]
Also #6512
